tests/Organization.py

```diff
@@ -741,6 +741,13 @@ def testRemoveCustomProperty(self):
         with self.assertRaises(github.UnknownObjectException):
             self.org.get_custom_property("property_1")
 
+    def testGetSelfHostedRunners(self):
+        # XXX: Organization BeaverSoftware does not exist
+        # https://api.github.com/orgs/BeaverSoftware
```

Comment:
```suggestion
```

Reply:
Sure, I think it could be done. I just wanted to justify why the BeaverSoftware couldn't be used

Reply:
Oh never mind, if you want to have those comments there, then that's fine. In this case, it's ready for your review. Thanks for your patience.